evaluator/remediations.go

```diff
@@ -13,7 +13,7 @@ import (
 
 var remediationsPublisher mqueue.Writer
 
-func init() {
```

Comment:
I don't know why, with init() it didn't work with a Clowder. It could not load KAFKA_ADDRESS env var here. So I've changed that to standard function calling it from evaluator.configure()